docs/how-to-guides/software-management/software-support.md

```diff
@@ -1,34 +1,52 @@
 # Software Support
-This document provides information about the software support process, including scheduling, project management tools and practices, issue soliciation, issue selection, and prioritization.
 
+!!! Note "How to Submit Issues" 
+    Our code is managed [via GitHub](https://github.com/DOI-USGS)
 
-## Step 1: Scheduling the Support Sprint
+    Any issues with our software portfolio should be tracked in our repos through [GitHub issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues). Links for some our bigger repos: 
+
+    * [ISIS](https://github.com/DOI-USGS/ISIS3/issues)
+    * [ALE](https://github.com/DOI-USGS/ale/issues)
+    * [USGSCSM](https://github.com/DOI-USGS/usgscsm/issues)
+
+    This document provides information about the software support process. That is, how our development team goes about addressing these issues. Including scheduling, project management tools and practices, issue soliciation, issue selection, and prioritization.
+
+## Support Sprint Process 
+
+Support sprints are dedicated efforts, time-boxed to 3 weeks quarterly. We schedule these as part of our internal scheduling procedures, and support sprints are scheduled 3 to 6 weeks ahead of time. This includes a prioritization meeting where we go through the issues accross our repositories and put them in prioritization order. These meetings are open to the public. 
```

Comment:
Replace: This includes a prioritization meeting where we go through the issues accross our repositories and put them in prioritization order. These meetings are open to the public.

With something like: A public prioritization meeting is scheduled prior to starting a support sprint. A notification of this meeting will be made (see below for how to subscribe) to the support mailing list 3 to 6 weeks ahead of time.